src/routes/institution-route.js

```diff
@@ -4,9 +4,11 @@ import * as controller from '../controllers/institution-controller.js'
 const institutionRouter = express.Router({ mergeParams: true })
 
 institutionRouter.get('/', controller.fetchProjectInstitutions)
+institutionRouter.get('/add', controller.searchInstitutionById)
```

Comment:
I mentioned in the web PR but this is confusing because the route is called `add` but it only searches and returns the value.